src/screens/ProfileScreen/index.js

```diff
@@ -272,9 +274,6 @@ const ProfileScreen = ({route}) => {
       const result = await getSelfFeedsInProfile(offset, limit);
       const {data: dataMyFeed} = result;
       const {mapNewData} = mappingColorFeed({dataFeed: dataMyFeed, dataCache: cacheFeed});
-      if (Array.isArray(result.data) && result.data.length === 0) {
-        setIsLastPage(true);
-      }
       if (offset === 0) {
         StorageUtils.myFeeds.set(JSON.stringify(mapNewData));
         setMyProfileFeed(mapNewData, myProfileDispatch);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The check for whether the result data array length is zero (which was previously used to set `isLastPage`) has been removed. Ensure that the new implementation correctly handles the case when there are no more feeds to fetch.

```diff
- if (Array.isArray(result.data) && result.data.length === 0) {
-   setIsLastPage(true);
- }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->